components/Form/Form.js

```diff
@@ -0,0 +1,188 @@
+                  <option value="Other">Other</option>
+                </Select>
+              </Label>
+            </SomeoneElse>
+          )}
+          <Label htmlFor="importance">
+            {" "}
+            Importance :
+            <Select name="importance">
+              <option defaultValue="High">High</option>
+              <option defaultValue="Medium">Medium</option>
+              <option defaultValue="Low">Low</option>
+            </Select>
+          </Label>
+          <Label htmlFor="medication_name">
+            {" "}
+            Medication Name :
+            <Input type="text" name="medication_name" />
+          </Label>
+          <Label htmlFor="activ_ingredients">
+            {" "}
+            Activ Ingredients :
+            <Input type="text" name="activ_ingredients" />
+          </Label>
+          <Label htmlFor="dosage">
+            {" "}
+            Dosage :
+            <Input
+              type="number"
+              name="dosage"
+              id="dosage"
+              min="0"
+              step="0.01"
+            />
+          </Label>
+          <Label htmlFor="reason">
+            {" "}
+            Reason for taking:
+            <Input type="text" name="reason" />
+          </Label>
+          <Label htmlFor="form">
+            {" "}
+            Form :
+            <Select name="form">
+              <option defaultValue="Tablet">Tablet</option>
+              <option defaultValue="Capsule">Capsule</option>
+              <option defaultValue="Syrup">Syrup</option>
+              <option defaultValue="Injection">Injection</option>
+              <option defaultValue="Powder">Powder</option>
+              <option defaultValue="Ointment">Ointment</option>
+              <option defaultValue="Syringe">Syringe</option>
+              <option defaultValue="Cream">Cream</option>
+              <option defaultValue="Drops">Drops</option>
+              <option defaultValue="Spray">Spray</option>
+              <option defaultValue="Other">Other</option>
+            </Select>
+          </Label>
+          <SomeoneElse>
+            <Label htmlFor="frequency">
+              {" "}
+              Frequency :
+              <Select name="frequencyType">
+                <option defaultValue="Daily">Daily</option>
+                <option defaultValue="Weekly">Weekly</option>
+                <option defaultValue="Monthly">Monthly</option>
+              </Select>
+              <Select name="frequencyTimes">
+                <option defaultValue="Once">Once</option>
+                <option defaultValue="Twice">Twice</option>
+                <option defaultValue="Thrice">Thrice</option>
+                <option defaultValue="Four times">Four times</option>
+                <option defaultValue="Five times">Five times</option>
+                <option defaultValue="Six times">Six times</option>
+                <option defaultValue="More">More</option>
+              </Select>
```

Comment:
```suggestion
              <Select name="frequencyType">
                <option value="Daily">Daily</option>
                <option value="Weekly">Weekly</option>
                <option value="Monthly">Monthly</option>
              </Select>
              <Select name="frequencyTimes">
                <option value="Once">Once</option>
                <option value="Twice">Twice</option>
                <option value="Thrice">Thrice</option>
                <option value="Four times">Four times</option>
                <option value="Five times">Five times</option>
                <option value="Six times">Six times</option>
                <option value="More">More</option>
              </Select>
```

And: every select element should have a label!